tnb/validators.py

```diff
@@ -2,15 +2,15 @@
 
 
 class Validator(BaseClient):
-    def fetch_accounts(self):
+    def fetch_accounts(self) -> object:
         """
         Fetch accounts from validator
         Return response as a Python object
         """
 
         return self.fetch("/accounts")
 
-    def fetch_account_balance(self, account_number):
+    def fetch_account_balance(self, account_number: str) -> object:
```

Comment:
This method returns a `dict`.

**Note**: The method documentation is also incorrect. 

```suggestion
    def fetch_account_balance(self, account_number: str) -> dict:
```